scripts/generate_grpc_stubs.py

```diff
@@ -11,8 +11,13 @@
 STUBS_NAMESPACE = "ni_measurementlink_service._internal.stubs"
 PROTO_PARENT_NAMESPACES = ["ni.measurementlink", "nidevice_grpc", "ni.protobuf.types"]
 STUBS_PATH = pathlib.Path(__file__).parent.parent / STUBS_NAMESPACE.replace(".", "/")
-PROTO_PATH = STUBS_PATH / "proto"
-PROTO_FILES = list(PROTO_PATH.rglob("*.proto"))
+PROTO_PATH = pathlib.Path(__file__).parent.parent / "third_party" / "ni-apis"
+STUBS_PROTO_PATH = STUBS_PATH / "proto"
+STUBS_PROTO_FILES = list(STUBS_PROTO_PATH.rglob("*.proto"))
+GRPC_DEVICE_PROTO_PATH = PROTO_PATH / "ni" / "grpcdevice" / "v1"
```

Comment:
Add a comment saying why this session.proto is still here, with the AB# of the bug or tech debt